test/functional/system/transactions/expire_transactions.js

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const liskElements = require('lisk-elements').default;
+const randomUtil = require('../../../common/utils/random');
+const accountsFixtures = require('../../../fixtures/accounts');
+const Scenarios = require('../../common/scenarios');
+const queriesHelper = require('../../common/sql/queriesHelper.js');
+const localCommon = require('../common');
+const Bignum = require('../../../../helpers/bignum.js');
+
+const multiSig = new Scenarios.Multisig();
+
+describe('expire transaction', () => {
+	let library;
+	let queries;
+	let memAccountsSnapshotBeforeUndo;
+
+	const { expiryInterval } = __testContext.config.transactions;
+	const unconfirmedTransactionTimeOut =
+		global.constants.unconfirmedTransactionTimeOut;
+	// override transaction expiryInterval and unconfirmedTransactionTimeOut
+	// to test undo unConfirmed expired transactions
+	before(done => {
+		__testContext.config.transactions.expiryInterval = 5000;
+		global.constants.unconfirmedTransactionTimeOut = 0;
```

Comment:
How will application behave in case of setting the value of `global.constants.unconfirmedTransactionTimeOut` to 0? Will it never expire transactions?